lib/vtls/mbedtls_threadlock.c

```diff
@@ -55,8 +55,8 @@ int Curl_mbedtlsthreadlock_thread_setup(void)
     return 0;     /* error, no number of threads defined */
 
   for(i = 0;  i < NUMT;  i++) {
-    int ret;
 #if defined(USE_THREADS_POSIX) && defined(HAVE_PTHREAD_H)
+    int ret;
     ret = pthread_mutex_init(&mutex_buf[i], NULL);
```

Comment:
:+1:, nice catch. It does beg the question though, shouldn't we just shorten it to:
```C
if (pthread_mutex_init(&mutex_buf[i], NULL))
  return 0;
```
..to increase readability?

Reply:
Seems reasonable to me!

Reply:
For consistency it would then be nice to eliminate all the ret variables in the file as none of them is used besides for checking call result. Should I update the pull request?

Reply:
> For consistency it would then be nice to eliminate all the ret variables in the file as none of them is used besides for checking call result. Should I update the pull request?

ok

Reply:
That would be great, let's see what that changeset would look like.

Reply:
I have updated the pull request with the suggested changes